.eslintrc.js

```diff
@@ -1,28 +1,13 @@
 module.exports = {
-  parser: '@typescript-eslint/parser',
-  extends: ['plugin:@typescript-eslint/recommended', 'plugin:react/recommended', 'react-app'],
-  plugins: ['@typescript-eslint', 'react'],
-  parserOptions: {
-    ecmaFeatures: {
-      jsx: true,
+  extends: ['plugin:@typescript-eslint/recommended'],
+  plugins: ['@typescript-eslint'],
+  overrides: [
+    {
+      files: ['*.test.ts'],
+      rules: {
+        '@typescript-eslint/no-var-requires': 0,
+        '@typescript-eslint/no-explicit-any': 0,
+      },
     },
-  },
-  settings: {
-    react: {
-      version: 'detect',
-    },
-  },
-  rules: {
-    '@typescript-eslint/indent': ['warn', 2],
```

Comment:
Please let me know if you'd like to keep any of these overrides. All code is now completely eslint-compliant after removing these exceptions. I assumed this config file was simply copy pasted from another (react) project.